net/iproute2/BUILD

```diff
@@ -2,9 +2,9 @@ if ! in_depends $MODULE db; then
   sedit 's/arpd//' misc/Makefile
 fi &&
 
-./configure &&
-make &&
-prepare_install &&
-make SBINDIR=/usr/sbin install &&
+OPTS+=" --sbindir=/usr/bin" &&
```

Comment:
To be fair, lnet has other problems besides a hard-coded path.  At one point it asks you for extra options to the "ifconfig" command.